lib/src/screens/add_task.dart

```diff
@@ -1,9 +1,8 @@
 import 'package:flutter/material.dart';
 
-// TODO: Replace categories to use Task class after Issue #3 is completed
-// TODO: Refactor styles to use global theming after more designer guidance
```

Comment:
I wouldn't delete these TODOs. Good to have so we don't forget about the changes we need to make to this file.

Reply:
Just updated this - I imagine it was a merge conflict issue